### PR TITLE
Add geo city per bug 1227278

### PR DESCRIPTION
**Pull request by mreid-moz**
r? @trink

cc @SamPenrose

### CODE REVIEW

File: hindsight/output/executive_summary_full.lua [hunk 1]
```diff
@@ -69,6 +69,7 @@ local schema = {
     {"documentId"                   ,"CHAR"             ,36     ,nil        ,"Fields[documentId]"},
     {"docType"                      ,"CHAR"             ,36     ,nil        ,function () return doc_type end},
     {"country"                      ,"VARCHAR"          ,5      ,nil        ,function () return fx.normalize_country(read_message("Fields[geoCountry]")) end},
+    {"city"                         ,"VARCHAR"          ,32     ,nil        ,"Fields[geoCity]"},
```
Comment (trink) [inline hindsight/output/executive_summary_full.lua]: Columns should only be added to the end

File: heka/sandbox/decoders/http_edge_decoder.lua [hunk 2]
```diff
@@ -35,19 +35,27 @@ local UNK_GEO = "??"
...
     end
-    return country or UNK_GEO
+    return geo or default_value
+end
+
+function get_geo_country(xff, remote_addr)
+    return get_geo_field(xff, remote_addr, "country_code", UNK_GEO)
+end
+
+function get_geo_country(xff, remote_addr)
```
Comment (trink) [inline heka/sandbox/decoders/http_edge_decoder.lua]: redefinition

File: heka/sandbox/decoders/http_edge_decoder.lua [hunk 3]
```diff
@@ -113,6 +121,7 @@ function process_message()
     local xff = landfill_msg.Fields["X-Forwarded-For"]
     local remote_addr = landfill_msg.Fields["RemoteAddr"]
     landfill_msg.Fields.geoCountry = get_geo_country(xff, remote_addr)
+    landfill_msg.Fields.geoCity = get_geo_city(xff, remote_addr)
```
Comment (trink) [inline heka/sandbox/decoders/http_edge_decoder.lua:124]: you can do city and country in one lookup

Comment (mreid-moz) [inline heka/sandbox/decoders/http_edge_decoder.lua:124]: I'm ok with looking the up one at a time

**Pull request merged**